src/views/EditTodo.js

```diff
@@ -0,0 +1,9 @@
+import React, { Component } from "react";
+
+export default class EditTodo extends Component {
```

Comment:
You can change it to functional component if you are not dealing with state.

Reply:
Okay..later when we connect it to redux it has to be class component so as to have its own state right ?

Reply:
Ok. But i am not expecting redux in this app.

Reply:
Ok